vector/src/Data/Vector/Generic.hs

```diff
@@ -382,13 +379,13 @@ unsafeLastM v = unsafeIndexM v (length v - 1)
 
 -- | /O(1)/ Yield a slice of the vector without copying it. The vector must
 -- contain at least @i+n@ elements.
-slice :: Vector v a => Int   -- ^ @i@ starting index
+slice :: (HasCallStack, Vector v a)
+                    => Int   -- ^ @i@ starting index
                     -> Int   -- ^ @n@ length
                     -> v a
                     -> v a
```

Comment:
```suggestion
      => Int   -- ^ @i@ starting index
      -> Int   -- ^ @n@ length
      -> v a
      -> v a
```
This way it's aligned with the `::`.

Reply:
lol. I did exactly the same thing then reverted back to minimize the diff.